lib/contentful_model/associations/belongs_to.rb

```diff
@@ -1,17 +1,22 @@
 module ContentfulModel
   module Associations
+    # Defines Belongs To association
     module BelongsTo
       def self.included(base)
         base.extend ClassMethods
       end
 
+      # Class method
```

Comment:
Hmm, that is a comment of limited use. Is there a way to make it more expressive/informative? Similar discussion could be started about the `Defines Belongs To association`: is there something we can use to add more value to the comments?